app/controllers/evaluations_controller.rb

```diff
@@ -32,68 +32,68 @@ def process_feedback
   def export_overall_course_rating
     if params[:provider].present? && params[:course_id].present?
       mooc_provider = MoocProvider.find_by!(name: params[:provider])
-      course = [Course.find_by!(provider_course_id: params[:course_id], mooc_provider: mooc_provider)].first()
+      course = [Course.find_by!(provider_course_id: params[:course_id], mooc_provider: mooc_provider)].first
```

Comment:
Why don't use: 'Course.find_by!(provider_course_id: params[:course_id], mooc_provider: mooc_provider)' ? This should return exactly one course (the first one in the database matching the criteria).